nozzle/api_client.go

```diff
@@ -41,17 +42,23 @@ func newAppInfo(app cfclient.App) *AppInfo {
 
 // NewAPIClient crate a new ApiClient
 func NewAPIClient(conf *NozzleConfig) (*APIClient, error) {
-	apiURL := conf.APIURL
+	apiURL := strings.Trim(conf.APIURL, " ")
```

Comment:
Fine for this PR. But going forward, whatever is needed in master, let's create a PR against master first. We can merge those changes into this branch from master. Also if anything from here is needed in master, let's create one this one time after this PR is merged.

Reply:
they are in other branch too...

https://github.com/wavefrontHQ/cloud-foundry-nozzle-go/commits/v1.0.1